tests/out/spv/globals.spvasm

```diff
@@ -39,43 +39,44 @@ OpDecorate %19 Block
 %16 = OpTypeStruct %15 %12
 %17 = OpTypeVector %12 2
 %18 = OpTypeRuntimeArray %17
-%19 = OpTypeStruct %18
-%21 = OpTypePointer Workgroup %14
-%20 = OpVariable  %21  Workgroup
-%23 = OpTypePointer Workgroup %6
-%22 = OpVariable  %23  Workgroup
-%25 = OpTypeStruct %16
-%26 = OpTypePointer StorageBuffer %25
-%24 = OpVariable  %26  StorageBuffer
-%28 = OpTypePointer StorageBuffer %19
-%27 = OpVariable  %28  StorageBuffer
+%20 = OpTypePointer Workgroup %14
+%19 = OpVariable  %20  Workgroup
+%22 = OpTypePointer Workgroup %6
+%21 = OpVariable  %22  Workgroup
+%24 = OpTypeStruct %16
+%25 = OpTypePointer StorageBuffer %24
+%23 = OpVariable  %25  StorageBuffer
+%27 = OpTypeStruct %18
+%28 = OpTypePointer StorageBuffer %27
+%26 = OpVariable  %28  StorageBuffer
 %30 = OpTypePointer Function %12
 %32 = OpTypePointer Function %4
 %35 = OpTypeFunction %2
 %36 = OpTypePointer StorageBuffer %16
 %37 = OpConstant  %6  0
-%40 = OpTypePointer Workgroup %12
-%41 = OpTypePointer StorageBuffer %12
-%42 = OpConstant  %6  1
-%45 = OpConstant  %6  3
-%47 = OpTypePointer StorageBuffer %15
-%48 = OpTypePointer StorageBuffer %12
-%52 = OpConstant  %6  256
+%39 = OpTypePointer StorageBuffer %18
```

Comment:
For what it's worth, this `%39` is a new type that's never used. I don't think it matters.